apis/collectioncsvapi/collectioncsvapi.yaml

```diff
@@ -0,0 +1,493 @@
+        Science-10,id1,5. Human Body,,,,This chapter describes about human body,Human Body,"head, eyes, nose, mouth",Yes,MC8L1V,do_31260042816321126413012,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+        Science-10,id2,5. Human Body,5.1 Parts of Body,,,"This section describes about various part of the body such as head, hands, legs etc.",Parts of Body,,No,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+        Science-10,id3,5. Human Body,5.1 Parts of Body,5.1.1 Key parts in the head,,,Parts of Head,,Yes,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+        Science-10,id4,5. Human Body,5.1 Parts of Body,5.1.2 Other parts,,,,,No,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+        Science-10,id5,5. Human Body,5.2 Organ Systems,,,,Organ Systems,,No,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,
+        Science-10,id6,5. Human Body,5.2 Organ Systems,5.2.1 Respiratory System,,,Respiratory System,,No,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,,"
+
+
+        The fields marked with an asterisk (*) are mandatory. They cannot be null or empty.'
+      operationId: TOCUpload
+      parameters:
+        - name: Content-Type
+          in: header
+          description: "The Content Type entity is the media type of the resource. Possible\
+            \ media types are:- \n  - Application/json\n  - Multipart/form-data\n  -\
+            \ Application/x-www-form-urlencoded"
+          required: true
+          schema:
+            type: string
+            example: 'application/json'
+        - name: X-Authenticated-User-Token
+          in: header
+          description: 'Specify authorization key (format: Bearer api-key) received from administrator when performing given API request.'
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '{{authToken}}'
+        - name: Authorization
+          in: header
+          description: Access token of registered user performing given API request.
+          required: true
+          style: simple
+          schema:
+            type: string
+            example: '{{api-key}}'
+        - name: collection-id
+          in: path
+          description: 'Collection id to upload. It should be unique and same as csv file identifier'
+          required: true
+          style: simple
+          schema:
+            type: string
+      requestBody:
+        content:
+          application/x-www-form-urlencoded:
+            encoding: {}
+            schema:
+              required:
+                - fileUrl
+              type: object
+              properties:
+                fileUrl:
+                  type: string
+                  description: 'File which user want to upload'
+        required: false
+      responses:
+        '200':
+          description: 'SUCCESS. The "TOC Upload" operation was successful!'
```

Comment:
@Jayaprakash8887 modify the message to ' SUCCESS. The TOC was uploaded successfully.'